#!/usr/bin/env python3

PACKAGE = "actor_lidar"

from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("center_offset", double_t, 0, "Offset (degrees) direction from front center", 0, -87.25, 87.25)
gen.add("field_of_view", double_t, 0, "Field of view (degrees) used for closest object detection", 30, 5, 180)
gen.add("min_range", double_t, 0, "Minimum range (m) for detection", 0.1, 0, 10)
gen.add("max_range", double_t, 0, "Maximum range (m) for detection", 10, 0, 10)

exit(gen.generate(PACKAGE, "lidar_2d_node", "Lidar2D"))
